fedora/jellyfin-web.spec

```diff
@@ -2,7 +2,7 @@
 
 Name:           jellyfin-web
 Version:        10.7.0
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
We don't need to change this against master, since it's not building real packages, and will just mess with the version bump later.